dice_ml/utils/helpers.py

```diff
@@ -90,6 +95,11 @@ def load_custom_testing_dataset():
     return pd.DataFrame(data, columns=['Categorical', 'Numerical', 'Outcome'])
 
 
+def load_custom_vars_testing_dataset():
```

Comment:
Shall we move this code to conftest.py? This shouldn't be a part of this package.

Reply:
Good point. I guess we can move some of the older functions also that are only used in test.

Reply:
Done.